stix_shifter_modules/tanium/configuration/config.json

```diff
@@ -0,0 +1,29 @@
+{
+    "connection": {
```

Comment:
few connection parameters are missing. such as -
"type": {
            "displayName": "Tanium",
            "group": "tanium"
        }

have a look at other modules to verify what's missing